courses/free-courses-vi.md

```diff
@@ -42,14 +44,17 @@
 * [Lập trình Android - Android Widgets - Các điều khiển](https://www.youtube.com/watch?v=2xDWeWllzWg&list=PLv6GftO355At6jjYThbMn-5r164GJ5Vyb) - ZendVN
 * [Lập trình Android - Menu - Context Menu - Dialog](https://www.youtube.com/watch?v=vS_InAnGUFA&list=PLv6GftO355Avjf5iuNbEUsIZbltzDEuIU) - ZendVN
 * [Lập trình Android - Xây dựng bố cục giao diện với Android Layout](https://www.youtube.com/watch?v=oxWDjizEN6g&list=PLv6GftO355AtfPQx7M3dkWgi9KPUB9S0V) - ZendVN
+* [Lập trình Android A-Z](https://www.youtube.com/watch?v=P60kcSaeFmg&list=PL5uqQAwS_KDjAgLGiaCakwJV1f4vRnTLS) - KhoaPham
```

Comment:
the corresponding playlist is https://www.youtube.com/playlist?list=PL5uqQAwS_KDjAgLGiaCakwJV1f4vRnTLS These are the kind of youtube links we prefer. I know other links on this page don't follow this rule; they should be changed.